backend/src/openbeheer/zaken/views.py

```diff
@@ -0,0 +1,92 @@
+import uuid
+
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema
+from msgspec.json import decode
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from openbeheer.api.views import MsgspecAPIView
+from openbeheer.clients import zrc_client
+from openbeheer.types import (
+    OBField,
+    OBFieldType,
+    OBList,
+    OBPagination,
+    OBSelection,
+    Zaak,
+    ZGWError,
+    ZGWResponse,
+    as_ob_option,
+)
+
+
+@extend_schema(
+    summary=_("Zaken"),
+)
+class ZakenView(MsgspecAPIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request: Request):
+        page = int(request.GET.get("page", 1))
+        page_size = 100  # TODO: magic number
+        client = zrc_client()
+        with client:
+            response = client.get(
+                "zaken",
+                params={
+                    "expand": ["status", "status.statustype"],
+                    "pageSize": page_size,
+                },
+            )
+
+        if not response.ok:
+            error = decode(response.content, type=ZGWError)
+            return Response(error, status=response.status_code)
+
+        data = decode(response.content, type=ZGWResponse[Zaak])
+
+        # XXX: Not filterable on API, should we even return OBOption to frontend?
+        statustypes = sorted(
+            {
+                as_ob_option(st)
+                for z in data.results
+                if (s := z._expand.status) and (st := s._expand.statustype)
+            },
+            key=lambda o: o.label,
+        )
+
+        resp = OBList(
```

Comment:
This feels like we can abstract this further so we only have minimal configuration for every resource? Also: aren't we re-implementing serializers here?

Reply:
> This feels like we can abstract this further so we only have minimal configuration for every resource?

Yes. And when the count reaches >= 3 and we see the real patterns emerge, we can abstract further.

> Also: aren't we re-implementing serializers here?

No, we're using an existing, modern, type annotation friendly (de-)serialisation library, with built-in camel case renaming.
If anything, we're re-implementing `zgw_consumers/api_models` here (or more precisely, in `openbeheer.types`. As this open issue maykinmedia/zgw-consumers/issues/51 suggests. See discussion on that issue.

So eventually, this will probably move upstream.

Reply:
Pleas discuss with @SilviaAmAm , I think DRF does camel case renaming out of the box as well and I think mixing DRF with non-DRF patterns is confusing. I'd rather use full DRF or full library X, I'm a little bit reluctant to build a hybrid.

Reply:
Don't get me wrong, I do like the typse and their setup, I do want to double check this so we set this up correctly for the future endpoints and I want to make sure we have something we're all (reasonable) comfortable working with.

> Yes. And when the count reaches >= 3 and we see the real patterns emerge, we can abstract further.

Fair enough 👍 , this makes it a little bit harder for me to see the bigger picture of what will go where though. If I later on want to add a an endpoint or resource, how much (or preferable little) do I need to add. 

Would be cool if we could do something close to `api_paths_from_resource("zaken")` which returns the paths and presto 😎

Reply:
As discussed: we leave it for now.